prettier.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  singleQuote: true,
+  trailingComma: 'all',
+	arrowParens: 'avoid',
```

Comment:
I think there is extra space in this line.

Reply:
How strange, there is no extra space in my code. But, I removed all the spaces and added them again.